### DISCUSSION TITLE
Are there shortcut for hover tooltip

### DISCUSSION THREAD

**Discussion by bazhenov** [zed-industries/zed#6489] [Feature Requests]
When hover over method or variable I get a tooltip with a documentation. Is there any keyboard shortcut for it?

**Comment by JosephTLyons** [comment]
A keybinding was added and will be available when 0.76.x hits stable (should be 3/15/2023):

<img width="561" alt="SCR-20230312-nagx" src="https://user-images.githubusercontent.com/19867440/224565361-e5f5d459-0b81-48ab-9378-da516bb85044.png">

**Comment by Warkanlock** [comment]
Is it possible to have the tooltip with information while we write the method's signature? Sometimes its easier to spot parameters that are required, summary comments that might be useful, etc, without needing to go to a ``hover`` command

**Reply by JosephTLyons** [reply]
I just wanted to mention that you don't need to use the hover command, as you can simply hover the mouse.  That being said, we don't currently have a way to show documentation tooltips as you type.

**Reply by Warkanlock** [reply]
Thanks for the quick update. I know Zed is not open source, although I can help implement this feature if required. 

So far, I'm almost about to get completed the migration from neovim to zed. Let's see if I can complete the transition by the EOM 

Great work🥇

**Reply by ghost** [reply]
@JosephTLyons Is there a command / keyboard shortcut to show type information of the element under the cursor (learning Rust). Currently using Vim mode.

**Comment by LeeviSF** [comment]
Related to this, is there keyboard shortcut to scroll the hover tooltip? Scrolling works with mouse.

**Reply by SomeoneToIgnore** [reply]
![image](https://github.com/zed-industries/zed/assets/2690773/69a90dbe-6446-4024-98cb-7eff142abf00)

There should be one, at least there's an action you can bind it to.

**Reply by LeeviSF** [reply]
That one opens the tooltip, mut if I want to scroll up/down the text in the tooltip, I need mouse for that.

**Reply by JosephTLyons** [reply]
Not that I'm aware of - feel free to open a new issue for that.

**Reply by Moshyfawn** [reply]
There's already a feature request for this #4787

**Reply by JosephTLyons** [reply]
Very nice catch! Thanks @Moshyfawn.

**Comment by matthiasg** [comment]
When I hover i get this with a tooltip saying " Claude Code: AI-powered coding assistance available here"  hovering over ConfigPaths 

<img width="1382" height="472" alt="CleanShot 2025-07-15 at 11 21 42@2x" src="https://github.com/user-attachments/assets/6ed83069-f899-495c-a2fc-05280dff6eed" />

I would have expected a tooltip with information about the type. Pressing Alt like in vscode does not do anything either nor do "go to definition"/"declaration". whats the cause of errors like these ? Is this related to these 'shortcuts' ?
